### PR TITLE
Add pranjal

### PR DESCRIPTION
**Pull request by Pranjalmishra30**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,15 +35,45 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
-### A
+### A  
+
+### C  
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
+- **How to sign up**: 
+    - [Hacktoberfest Website](https://hacktoberfest.digitalocean.com)
```
Comment (crweiner) [inline README.md:46]: No need to add the Hacktoberfest signup to here as it'll be redundant. 👍

File: README.md [hunk 2]
```diff
@@ -35,15 +35,45 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
 ## A to Z Order of Companies
 
-### A
+### A  
+
+### C  
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
```
Comment (crweiner) [inline README.md:44]: Can you please make this a hyperlink to their repos?

File: README.md [hunk 3]
```diff
@@ -35,15 +35,45 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
 
 ### 1 Merged PR
 
-### 2 or more PRs
+### 2 or more PRs  
+
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
```
Comment (crweiner) [inline README.md:67]: Double check the order of things here. 😄 The A->Z and Least -> Most Involvement sections have a different order. Check the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) file and [last year's branch](https://github.com/crweiner/hacktoberfest-swag-list/tree/2020) for examples of how to set this up.

**Review by crweiner — COMMENTED** [review]
Some things to change for consistency sake. Be sure to review the [Contributing.md file!](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md)

**Comment by crweiner** [comment]
Hey @Pranjalmishra30! 👋 Thanks so much for adding Camunda to the Hacktoberfest Swag List! 🎉 

There's a few things I'd like to give you the opportunity to change. Be sure to look at the [Contributing.md file](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) and [last year's branch](https://github.com/crweiner/hacktoberfest-swag-list/tree/2020) for examples of how to format the Swag List.

Additionally, please make the robots happy by tackling the issues [flagged by CodeClimate](https://codeclimate.com/github/crweiner/hacktoberfest-swag-list/pull/389). It's just trying to make your markdown better and make sure that the mardown -> HTML conversion goes well. 😄 

Thanks again and let me know if you have any questions!! 👍

File: README.md (hunk 3 above)
Comment (Pranjalmishra30) [reply README.md:67]: Oh my bad, actually both prizes are for the same level of involvement. I should have written 'T-shirt and stickers'.

File: README.md (hunk 2 above)
Comment (Pranjalmishra30) [reply README.md:44]: They have multiple repositories that one can contribute to, so i have only added the main one.

**Comment by Pranjalmishra30** [comment]
@crweiner Updated all changes as requested. Apologies for the multiple commits. 
Please let me know if there any other things that need to be changed.
Thanks :)

**Comment by Pranjalmishra30** [comment]
@crweiner Updated all changes as requested. Apologies for the multiple commits. Also there seems to be a [merge conflict](https://github.com/crweiner/hacktoberfest-swag-list/pull/389/conflicts), so do you want me to deal with it. Please let me know if there any other things that need to be changed.
Thanks :)

**Comment by crweiner** [comment]
Hey @Pranjalmishra30! Thank you so much for taking the time to add Camunda to the 2021 Hacktoberfest Swag List!! 🎉 🎉 Also, thank you for updating your PR and for following the guidelines! Super happy to get this merged and on the live list. 

Thanks again, and have a wonderful Hacktoberfest!

party time

**Pull request merged**